products/products.yaml

```diff
@@ -13,7 +13,7 @@ products:
     manifestsDir: "integreatly-3scale"
     quayScan: true
   - name: cloud-resource-operator
-    version: v0.42.0
+    version: v0.43.0
```

Comment:
we need to bump in `installation-cpaas.yaml` as well
https://github.com/integr8ly/integreatly-operator/blob/master/products/installation-cpaas.yaml#L68